client/app/telecom/pack/slots/hubic/pack-hubic.controller.js

```diff
@@ -27,14 +48,28 @@ angular.module("managerApp").controller("PackHubicCtrl", function (OvhApiPackXds
                     }
                 );
             });
-            return self.services;
+
+            return $q.all(_.map(_.filter(self.services, { isUsed: true }), function (service) {
+                return getVoucherDetails(service.domain);
+            })).then(function (result) {
+                _.times(result.length, function (index) {
+                    self.services[index].email = result[index].result.email;
+                });
+            }).catch(function (error) {
+                Toast.error([$translate.instant("pack_xdsl_hubic_domain_details_error"), error.error]).join(" ");
+                return $q.reject(error);
+            }).finally(function () {
+                Poller.kill({
+                    scope: $scope.id
```

Comment:
You can avoid using `$scope` by using `namespace` attribute on `Poll` declaration.
For example : 
https://github.com/ovh-ux/ovh-manager-telecom/blob/f67644af79ae38c78061c5bd8b45210d93c98be0/client/components/telecom/alias/hunting/sounds/telecom-telephony-alias-hunting-sounds.controller.js#L50-L58

But... You probably still need `$scope` to kill the polling at `$destroy` phase...